Sources/ClientRuntime/Networking/Http/URLSession/FoundationStreamBridge.swift

```diff
@@ -41,21 +41,37 @@ class FoundationStreamBridge: NSObject, StreamDelegate {
     /// A Foundation `OutputStream` that will read from the `ReadableStream`
     private let outputStream: OutputStream
 
-    /// Actor used to isolate the stream status from multiple concurrent accesses.
-    actor ReadableStreamStatus {
+    /// A Logger for logging events.
+    private let logger: LogAgent
```

Comment:
I put a Logger in this type, and I also fixed the issue where stream writes can get out-of-order.